lib/doc/worksheet.js

```diff
@@ -782,8 +783,32 @@ class Worksheet {
     this._media = value.media.map(medium => new Image(this, medium));
     this.sheetProtection = value.sheetProtection;
     this.tables = value.tables.reduce((tables, table) => {
-      const t = new Table();
-      t.model = table;
+      /**
+       * Once we have the table information and the sheet we can add the rows related to it.
+       * We parse the start and end via the reference and then we iterate over all the
+       * cells the sheet has to see if they are inside the table to add their values to its rows.
+       */
+      const [,startColumnStr, startRowStr, endColumnStr, endRowStr] = /(\D+)(\d+):(\D+)(\d+)/i.exec(table.tableRef);
+      const startColumn = startColumnStr.toUpperCase().charCodeAt(0) - 64;
+      const endColumn = endColumnStr.toUpperCase().charCodeAt(0) - 64;
+      const startRow = parseInt(startRowStr, 10);
+      const endRow = parseInt(endRowStr, 10);
+      const offset = table.headerRow ? 1 : 0;
+      const rows = [];
+
+      for(let i = (startRow + offset); i <= endRow; i++){
+        const rowValues = [];
+        that.getRow(i).eachCell((cell, colNumber) =>{
+          if(colNumber >= startColumn && colNumber <= endColumn){
+            rowValues.push(cell.value);
+          }
+        });
```

Comment:
The `rows` array is not being updated with the found row values.
I think you meant to put `rows.push(rowValues);` after the loop function.